plugins/UM3NetworkPrinting/src/ClusterUM3OutputDevice.py

```diff
@@ -66,10 +66,11 @@ def __init__(self, device_id, address, properties, parent = None) -> None:
         self._received_print_jobs = False # type: bool
 
         if PluginRegistry.getInstance() is not None:
-            self._monitor_view_qml_path = os.path.join(
-                PluginRegistry.getInstance().getPluginPath("UM3NetworkPrinting"),
-                "resources", "qml", "MonitorStage.qml"
-            )
+            plugin_path = PluginRegistry.getInstance().getPluginPath("UM3NetworkPrinting")
```

Comment:
Where is this stuff coming from?

Reply:
`PluginRegistry.getPluginPath()` returns `Optional[str]`, and I got a complain from mypy about this because it's used in `os.path.join()`.